### ISSUE TITLE
Option to re-order box plots?

### ISSUE THREAD

**Issue by ldecicco-USGS** [USGS-R/toxEval#280]
As mentioned in #268 it might be nice to see the Chemical boxplots sorted by chemical only (not chemical class). This would require an additional argument, and thought about how/if to incorporate that into the app.

**Comment by srcorsi-USGS** [comment]
If you are referring to resorting by just chemical as an option (radio-button), that would be fine, but I prefer to keep the capability to group them by chemical class. I think most people will prefer sorting them by class as a way to organize the analysis with a bit more structure.

**Comment by ldecicco-USGS** [comment]
Yeah, we would NOT lose the capacity to group by class, and that would remain the default. The question is just do we want to give an option to sort the chemicals based purely on EAR.

**Comment by srcorsi-USGS** [comment]
OK, I can see how it might be interesting for some people to see that, but grouping by class will be much more useful. I don't see it as a high priority item for now.
